assets/js/feature-tours/idea-hub-module.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Idea Hub module Tour.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+import { EVENTS } from 'react-joyride';
+
+/*
+ * Internal dependencies
+ */
+import { VIEW_CONTEXT_DASHBOARD } from '../googlesitekit/constants';
+import { CORE_MODULES } from '../googlesitekit/modules/datastore/constants';
+import { MODULES_IDEA_HUB } from '../modules/idea-hub/datastore/constants';
+
+const ideaHubModule = {
+	slug: 'ideaHubModule',
+	contexts: [ VIEW_CONTEXT_DASHBOARD ],
+	// TODO: change this if launch version for the feature changes.
+	version: '3.6.0',
+	checkRequirements: async ( registry ) => {
+		await registry.__experimentalResolveSelect( CORE_MODULES ).getModules();
+		const isIdeaHubModuleActive = registry.select( CORE_MODULES ).isModuleActive( 'idea-hub' );
+		const isIdeaHubModuleConnected = registry.select( CORE_MODULES ).isModuleConnected( 'idea-hub' );
+		const newIdeas = registry.select( MODULES_IDEA_HUB ).getNewIdeas() || [];
+
+		return ( isIdeaHubModuleActive && isIdeaHubModuleConnected && !! newIdeas.length );
+	},
+	steps: [
+		{
+			target: '.googlesitekit-idea-hub__title',
+			title: __( 'Get inspiration for new topics to write about!', 'google-site-kit' ),
+			content: __( 'These ideas are based on unanswered searches related to the content of your site. They are organised by topics and will refresh every N days.', 'google-site-kit' ),
+		},
+		{
+			target: '.googlesitekit-idea-hub__actions--create',
+			title: __( 'Start a draft', 'google-site-kit' ),
+			content: __( 'Found an interesting idea you want to write about? Create a draft! You can always get back to it later on the Posts page', 'google-site-kit' ),
+		},
+		{
+			target: '.googlesitekit-idea-hub__idea--single',
+			title: 'Save for later or dismiss',
+			content: "If you're not ready to create a draft about an idea just yet, add it to your 'Saved' list and revisit later. If you don’t like an idea, you can dismiss it from your list.",
```

Comment:
Is this about me forgetting to add `__( … )` rather than the types of quotation? I think it needs to use double quotes due to the words like `you're` and `don't`, no?

Reply:
>  I think it needs to use double quotes due to the words like you're and don't, no?

We use the special version of single quotes in our codebase: `’`. My suggestion adds it and changes double quotes on single quotes for the string itself. But you also right, you forgot to add `__( ... )` to these strings. Please, updated it too.

Reply:
💡 I see, thank you